scm-core/src/main/java/sonia/scm/web/UserAgent.java

```diff
@@ -159,6 +175,10 @@ public boolean isBrowser()
     return browser;
   }
 
+  public boolean isScmClient() {
+    return scmClient;
+  }
+
```

Comment:
A JavaDoc for this one like for the other ones might be useful. It might be useful to explain, what we meen with "scm client".